src/components/feedback/Popover.tsx

```diff
@@ -378,16 +424,12 @@ export default function Popover({
     <div
       className={classnames(
         'absolute z-5',
-        variant === 'panel' && [
-          'max-h-80 overflow-y-auto overflow-x-hidden',
-          'rounded border bg-white shadow hover:shadow-md focus-within:shadow-md',
-        ],
```

Comment:
The outermost Popover element is now "unstyled", and there's a new inner element wrapping children which is the one receiving styling classes.

This allows the arrow to be displayed inside the "unstyled" container, and keep the other one looking as before.

There's one consideration. The `classes` prop, which intuitively one would expect to be added to the outermost element, is now added to the "styled" inner element, to ensure it keeps looking the same. Maybe we should document this in the prop and in future we can add a new `containerClasses` prop if we need to provide classes to the outermost element.